server/src/app/auth/jwt.strategy.ts

```diff
@@ -0,0 +1,22 @@
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { AuthService } from './auth.service';
+import { PassportStrategy } from '@nestjs/passport';
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy) {
+    constructor(private readonly authService: AuthService) {
+        super({
+            jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+            secretOrKey: 'secretKey',
+        });
+    }
+
+    async validateToken(address: string) {
+        const user = await this.authService.createToken(address);
+        if (!user) {
```

Comment:
i'm not understanding the validation process.

when !user become true?

user is always definded if address is defined.